packages/docs/docs/billing/insurance-eligibility-checks.md

```diff
@@ -0,0 +1,135 @@
+
+The `item` field also provides additional data about the procedure, product, or service.
+
+| **Property**       | **Description**                                                                                                                                                   | **Code System**                                         | **Example**                   |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- | ----------------------------- |
+| `productOrService` | The product, drug, service, etc. that is being provided.                                                                                                          | [CPT Codes](https://www.ama-assn.org/topics/cpt-codes)  | 92340 - Fitting of eyeglasses |
+| `diagnosis`        | The diagnosis for which care is being sought.                                                                                                                     | [ICD-10 Codes](https://www.icd10data.com/ICD10CM/Codes) | Condition/reduced-vision      |
+| `category`         | The general type of the service or product being checked for eligibility.                                                                                         |                                                         | Vision Coverage               |
+| `provider`         | A reference to the [`Practitioner`](/docs/api/fhir/resources/practitioner) who is responsible for providing the service.                                          |                                                         | Practitioner/dr-alice-smith   |
+| `quantity`         | The number of repetitions of the service that will be performed.                                                                                                  |                                                         | 2                             |
+| `unitPrice`        | The price charged to the patient for a single unit of the service. This is the price that the provider charges for the service.                                   |                                                         | $200                          |
+| `facility`         | A reference to the [`Location`](/docs/api/fhir/resources/location) or [`Organization`](/docs/api/fhir/resources/organization) where the service will be provided. |                                                         | Organization/example-hospital |
+| `detail`           | A reference to the [`CarePlan`](/docs/api/fhir/resources/careplan) with details describing the service.                                                           |                                                         | CarePlan/improve-vision       |
+
+<details><summary>Example: A coverage eligibility request for a basic consultation</summary>
```

Comment:
We just upgraded to a newer version of Docusaurus which is more strict about MDX parsing. 

To get this to build, you should add the `<summary>` to a new line